jmeter-java-dsl/src/test/java/us/abstracta/jmeter/javadsl/core/BuildTreeContextTest.java

```diff
@@ -0,0 +1,55 @@
+package us.abstracta.jmeter.javadsl.core;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.LinkedHashMap;
+
+public class BuildTreeContextTest {
+
+    @Test
+    public void setEntryAddsKeyValuePair() {
+        BuildTreeContext context = new BuildTreeContext();
+        String key = "key";
+        String value = "value";
+
+        context.setEntry(key, value);
+
+        assertEquals(value, context.getEntry(key));
+    }
+
+    @Test
+    public void buildRemoteExecutionContextInitializesMaps() {
```

Comment:
buildRemoteExecutionContext should never be used by DSL end users, is quite simple, and only used in one place. I think is not worth adding a test currently (if we ever target to some particular coverage, then it would, but right now we don't).